pkgs/applications/virtualization/containerd/default.nix

```diff
@@ -4,15 +4,15 @@ with lib;
 
 buildGoPackage rec {
   pname = "containerd";
-  version = "1.4.2";
+  version = "1.4.3";
   # git commit for the above version's tag
   commit = "7ad184331fa3e55e52b890ea95e65ba581ae3429";
```

Comment:
Shouldn't this be changed if the version is changing?

If this is unused, should be removed

Reply:
This is passed into buildFlags, so yeah, this probably should've been updated. Will push a fix asap.

Reply:
Fixed in 2d55b8def4b8e4882123cae432bdea2c89e72166